rubygem-foreman_openscap/rubygem-foreman_openscap.spec

```diff
@@ -105,13 +106,14 @@ mkdir -p %{buildroot}%{foreman_bundlerd_dir}
 
 %posttrans
 # We need to run the db:migrate (because of SCAPtimony) after the install transaction
-/usr/sbin/foreman-rake db:migrate  >/dev/null 2>&1 || :
-/usr/sbin/foreman-rake db:seed  >/dev/null 2>&1 || :
-/usr/sbin/foreman-rake apipie:cache  >/dev/null 2>&1 || :
-(/sbin/service foreman status && /sbin/service foreman restart) >/dev/null 2>&1
+%foreman_db_migrate
```

Comment:
Ack, cancel that.. there are in scaptimony aren't there?  So leave this line in, but the others need to stay, or be replaced by macros if you like.